### ISSUE TITLE
Bad Performance in test data set

### ISSUE THREAD

**Issue by LiangSun1994** [cooijmanstim/recurrent-batch-normalization#7]
I have read your [this paper](https://arxiv.org/pdf/1603.09025.pdf) which give excellent result than original lstm. Recently, after implementing this idea in TensorFlow,  However, I have got very bad results in the test data set.  Here is the [source code](https://github.com/LiangSun1994/BN-LSTM).
The following picture is the result of accuracy.  The yellow line is training loss.   The blue line is training loss when the training flag in the batch norm is set to false. The red line is test loss.
![image](https://user-images.githubusercontent.com/50352879/78973939-712cad00-7b43-11ea-9706-3c027e0990cb.png)
I have tried the bn-lstm code you have mentioned. For example:
https://github.com/OlavHN/bnlstm
https://gist.github.com/spitis/27ab7d2a30bbaf5ef431b4a02194ac60
They got a similar result. Please take a look at this [discussion](https://github.com/OlavHN/bnlstm/issues/2). Please help me whether I have mad a stupid mistake.
@cooijmanstim @ballasn @Thrandis @caglar
